### ISSUE TITLE
position is not updated when changed in MarkerMixin.

### ISSUE THREAD

**Issue by osiloke** [nilpath/riot-google-maps#10]
This is due to the applyupdater not being able to compare two positions correctly. If you skip the equality check for just position updates or compare the lat and lng values, you will have a quick workaround i.e

```
if (updaterName == 'position' && (opt.lat() != prevOpt.lat() || opt.lng() != prevOpt.lng())){ 
      updater(opt, tag);
    } else if(!equals(opt, prevOpt)) {
      updater(opt, tag);
    }
```

**Comment by nilpath** [comment]
Yes, you're right, I think I only expected to compare lng / lat as regular objects and not instances of google.maps.LngLat

Not sure when I can get to fix this so if you feel up for it, you're more than welcome to make a PR :)
